sampleSpecs/misc.smithy

```diff
@@ -1,6 +1,8 @@
 namespace smithy4s.example
 
 use smithy4s.api#untagged
+use smithy4s.meta#vector
+use smithy4s.meta#indexedSeq
```

Comment:
Are these used in this file?

Reply:
Addressed in [b3365de](https://github.com/disneystreaming/smithy4s/pull/303/commits/b3365de87c6af31653ec9810054c9d128c6a0e07)